assets/dev/js/editor/editor-base.js

```diff
@@ -1298,8 +1297,14 @@ export default class EditorBase extends Marionette.Application {
 
 				let direction = 'max';
 
-				if ( 'desktop' !== device ) {
+				if ( 'desktop' === device ) {
+					desktopAppeared = true;
+				} else {
 					direction = activeBreakpoints[ device ].direction;
+
+					if ( desktopAppeared ) {
```

Comment:
If there will be 2 breakpoints that are greater then the desktop?

Reply:
> If there will be 2 breakpoints that are greater then the desktop?

It will still work. This change purpose is to check whether the `desktop` device already appeared in the loop, and if it is - we can start adding parents to the next devices control. The result is that every device which is geater than desktop, will not have a parent or child responsive controls.